api/v1beta1/elfcluster_types.go

```diff
@@ -26,6 +26,10 @@ const (
 	// resources associated with ElfCluster before removing it from the
 	// API server.
 	ClusterFinalizer = "elfcluster.infrastructure.cluster.x-k8s.io"
+
+	// ElfClusterForceDeleteAnnotation is an annotation that can be applied to ElfCluster
```

Comment:
// ElfClusterForceDeleteAnnotation means to skip the deletion of infrastructure resources in Tower (e.g. VM and labels)
// when deleting an ElfCluster. This is useful when the Tower server or SMTX ELF cluster is disconnected.